repose-aggregator/core/repose-core-api/src/main/resources/META-INF/schema/system-model/system-model.xsd

```diff
@@ -275,13 +283,153 @@
+            </xs:documentation>
+        </xs:annotation>
+        <xs:complexContent>
+            <xs:extension base="mod:FilterCriterion">
+                <xs:attribute name="regex" type="xs:string" use="required">
+                    <xs:annotation>
+                        <xs:documentation>
+                            <html:p>
+                                Regex defining what URI's this filter pertains too.
+                            </html:p>
+                        </xs:documentation>
+                    </xs:annotation>
+                </xs:attribute>
+            </xs:extension>
+        </xs:complexContent>
+    </xs:complexType>
+    <xs:complexType name="Not">
+        <xs:annotation>
+            <xs:documentation>
+                <html:p>Defines the logical NOT conditional processing element.</html:p>
+            </xs:documentation>
+        </xs:annotation>
+        <xs:complexContent>
+            <xs:extension base="mod:FilterCriterion">
+                <xs:choice>
+                    <xs:element name="method" type="mod:Method"/>
+                    <xs:element name="header" type="mod:Header"/>
+                    <xs:element name="uri" type="mod:Uri"/>
+                    <xs:element name="and" type="mod:And"/>
+                    <xs:element name="or" type="mod:Or"/>
+                </xs:choice>
+            </xs:extension>
+        </xs:complexContent>
+    </xs:complexType>
+    <xs:complexType name="And">
+        <xs:annotation>
+            <xs:documentation>
+                <html:p>Defines the logical AND conditional processing element.</html:p>
+            </xs:documentation>
+        </xs:annotation>
+        <xs:complexContent>
+            <xs:extension base="mod:FilterCriterion">
+                <xs:choice minOccurs="2" maxOccurs="unbounded">
+                    <xs:element name="method" type="mod:Method"/>
+                    <xs:element name="header" type="mod:Header"/>
+                    <xs:element name="uri" type="mod:Uri"/>
+                    <xs:element name="not" type="mod:Not"/>
+                    <xs:element name="or" type="mod:Or"/>
+                </xs:choice>
+            </xs:extension>
+        </xs:complexContent>
+    </xs:complexType>
+    <xs:complexType name="Or">
+        <xs:annotation>
+            <xs:documentation>
+                <html:p>Defines the logical OR conditional processing element.</html:p>
+            </xs:documentation>
+        </xs:annotation>
+        <xs:complexContent>
+            <xs:extension base="mod:FilterCriterion">
+                <xs:choice minOccurs="2" maxOccurs="unbounded">
+                    <xs:element name="method" type="mod:Method"/>
+                    <xs:element name="header" type="mod:Header"/>
+                    <xs:element name="uri" type="mod:Uri"/>
+                    <xs:element name="and" type="mod:And"/>
+                    <xs:element name="not" type="mod:Not"/>
```

Comment:
Or should be in this list.

Reply:
Actually, i'm not sure it is. i can't think of a case where it would make a difference, other than in how one thinks about it. Same for and i suppose.